### PR TITLE
Add language to ensure each Input Descriptor Object refers to only one container

### PR DESCRIPTION
**Pull request by wyc**
- In reference to: #27

### CODE REVIEW

File: spec/spec.md [hunk 1]
```diff
@@ -1314,6 +1314,15 @@ processing steps. Additional verification of credential data or subsequent
 validation required by a given Verifier are left to the Verifier's systems, code 
 and business processes to define and execute.
 
+During validation, each Input Descriptor Object ****MUST**** refer to only a
+singular presentation within a _Presentation Submission_, such that all checks
```
Comment (glcohen) [inline spec/spec.md:1318]: should presentation here be container? I think the terminology is confusing. On one hand a presentation submission contains a presentation, but the understanding here is that it contains multiple presentations.

Comment (wyc) [reply spec/spec.md:1318]: Yeah, this kinda highlights the importance of #90. I couldn't find a good word to express "things protected by the same signature within the same hierarchical grouping". Let's avoid "presentation" due to its overloading, I'll use "container" only. Does that sound reasonable?

Comment (glcohen) [reply spec/spec.md:1318]: sounds reasonable. we can always amend after #90 is settled.

Comment (wyc) [reply spec/spec.md]: Please let me know if the latest commit fixes your concern! Thanks

**Review by glcohen — APPROVED** [review]

**Pull request merged**